recommendation/views/images.py

```diff
@@ -0,0 +1,42 @@
+from urllib.parse import urlencode
+
+import requests
+from flask import abort, Blueprint, request, Response, stream_with_context
+
+from recommendation import conf
+
+EMBEDLY_RESIZE = 'https://i.embed.ly/1/display/resize'
+
+
+images = Blueprint('images', __name__)
+
+
+def make_embedly_url(**kwargs):
+    if 'url' not in kwargs:
+        raise RuntimeError()
+    qs = {}
+    for param in ['url', 'width', 'height']:
+        if param in kwargs:
+            qs[param] = kwargs[param][0]
+    qs['animate'] = False
+    qs['compresspng'] = conf.EMBEDLY_API_KEY
```

Comment:
@chuckharmston copy/paste error, either this should be True or a different config value.